src/main/java/employee/controller/EmployeeController.java

```diff
@@ -46,6 +78,23 @@
         return new ResponseEntity<>(HttpStatus.NOT_FOUND);
     }
 
+
+//    @ResponseBody
+//    @RequestMapping(value = "/employees/{id}", method = RequestMethod.GET, produces = "application/vnd.pl.employee+json;version:2.0")
+    public ResponseEntity<EmployeeMongo> getEmployeeV2(String employeeId) {
```

Comment:
It might make sense to name these methods as V1 and V2 but in reality a developer looking at it won't understand the difference between them. Rename the methods appropriately according to the business logic behind why we have 2 versions.